lib/src/Uncertainty/Algorithm/MetaModel/Kriging/openturns/KrigingEvaluation.hxx

```diff
@@ -84,7 +84,7 @@ public:
 protected:
 
   // BasisCollection ==> Persistent for save attribut
-  BasisPersistentCollection basis_;
+  Basis basis_;
```

Comment:
we need backward-compatible save/load methods

Reply:
Any example on how to implement it?

Reply:
yes, see OrthogonalUniVariatePolynomial:
```
void OrthogonalUniVariatePolynomial::load(Advocate & adv)
{
  UniVariatePolynomialImplementation::load(adv);
  // recurrenceCoefficients_ changed type from PersistentCollection<Coefficients> to Sample in 1.19
  // without backward compatibility, see https://github.com/openturns/openturns/pull/1961
  if (adv.getStudyVersion() >= 102000)
    adv.loadAttribute("recurrenceCoefficients_", recurrenceCoefficients_);
  else
  {
    PersistentCollection<Coefficients> coefficientsColl;
    adv.loadAttribute("recurrenceCoefficients_", coefficientsColl);
    const UnsignedInteger size = coefficientsColl.getSize();
    recurrenceCoefficients_ = PersistentCollection<Scalar>(3 * size);
    for (UnsignedInteger i = 0; i < size; ++ i)
      for (UnsignedInteger j = 0; j < 3; ++j)
        recurrenceCoefficients_[3 * i + j] = coefficientsColl[i][j];
  }
```

Reply:
Ok I see the point, thanks
There will be however some "lost" because the oldest implementation allowed basis collection to be wih different sizes (and it was bogus)